kimchi/src/circuits/polynomials/range_check/mod.rs

```diff
@@ -0,0 +1,97 @@
+//! Range check gate
+
+///```text
```

Comment:
Can you move this comment into the `circuitgates.rs` file?

Reply:
Done!